pkg/skaffold/event/v2/metadata.go

```diff
@@ -82,7 +82,8 @@ func getArtifacts(b latestV1.BuildConfig) []*proto.BuildMetadata_Artifact {
 	result := []*proto.BuildMetadata_Artifact{}
 	for _, a := range b.Artifacts {
 		artifact := &proto.BuildMetadata_Artifact{
-			Name: a.ImageName,
+			Name:    a.ImageName,
+			Context: a.Workspace,
```

Comment:
`a.Workspace` might just be `.` or a relative path if not set explicitly in the root config. If that's alright, then lgtm.

Reply:
I think that's alright, I can speak further with IDE folks regarding this